content/post/know-your-aws-community-day-india-2020-speaker-swapnil-jakhetia.md

```diff
@@ -0,0 +1,20 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T14:12:08Z
+description = "Swapnil Jakhetia will be speaking on \"Infrastructure is Code with AWS CDK\" at <time> on <date> at AWS Community Day India 2020."
+feature_image = "/images/swapnil.PNG"
```

Comment:
I think we will need a slightly smaller featured image because the current one crops off edges

![image](https://user-images.githubusercontent.com/25424/94035598-f6431000-fdcb-11ea-9139-19d05016d080.png)

Reply:
Corrected.

Reply:
Session/virtual edition is still getting cut off